doc/nrf/links.txt

```diff
@@ -858,6 +858,7 @@
 .. _`Verifying Fast Pair`: https://developers.google.com/nearby/fast-pair/help#verifying_fast_pair
 .. _`Fast Pair Personalized Name extension`: https://developers.google.com/nearby/fast-pair/specifications/extensions/personalizedname
 .. _`Fast Pair Certification Guidelines for Personalized Name`: https://developers.google.com/nearby/fast-pair/certification-guideline#2_personalized_name
+.. _`Fast Pair Battery Notification extension`: https://developers.google.com/nearby/fast-pair/specifications/extensions/batterynotification#BatteryNotification
```

Comment:
`https://developers.google.com/nearby/fast-pair/specifications/extensions/batterynotification`
There is not need to link directly to the section